app/views/components/summaryList.scala.html

```diff
@@ -35,7 +37,12 @@
             <a class="govuk-link" href="@summary.rows(fullstop).actions.head.items.head.href">
                 <span class="govuk-visually-hidden">
                     @summary.rows(fullstop).actions.head.items.head.visuallyHiddenText</span>
-                <span> @Html(messages("cf.cash-account.detail.csv.download"))</span>
+                <span>
```

Comment:
We can simplify the code to avoid repetition especially the `Html` helper
```suggestion
<span>
    @Html(messages(if (change) "cf.cash-account.transactions.request.review.change" else "cf.cash-account.detail.csv.download"))
</span>     
```